src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/QuestionSteps.java

```diff
@@ -73,38 +79,53 @@ public void cleanUp() {
             questionRepository.deleteById(questionId);
         }
 
-        OnlineHearing onlineHearing = testContext.getScenarioContext().getCurrentOnlineHearing();
-        if (onlineHearing != null) {
-            try {
-                onlineHearingPanelMemberRepository.deleteByOnlineHearing(onlineHearing);
-                onlineHearingRepository.deleteByCaseId(onlineHearing.getCaseId());
-            } catch(DataIntegrityViolationException e){
-                log.error("Failure may be due to foreign key. This is okay because the online hearing will be deleted elsewhere." + e);
-            }
+        try {
+            String onlineHearingExternalRef = testContext.getScenarioContext().getCurrentOnlineHearing().getCaseId();
+            onlineHearingPanelMemberRepository.deleteByOnlineHearing(onlineHearing);
+            onlineHearingRepository.deleteByCaseId(onlineHearingExternalRef);
+        } catch(DataIntegrityViolationException e){
+            log.error("Failure may be due to foreign key. This is okay because the online hearing will be deleted elsewhere." + e);
         }
     }
 
-    @And("^the draft a question for online_hearing$")
-    public void theDraftAQuestion() throws Throwable {
+    @And("^the post request is sent to create the question and the response status is (\\d+)$")
```

Comment:
I find this kind of semantics too complicated and harder to re-use. It's fine for now e.g. I want to refactor some more so that checking the response is one step that can be shared with other steps via the TestContext.

This is fine for now though. Fine to refactor when you have time.